src/traversal/printRelayOSSQuery.js

```diff
@@ -121,10 +121,8 @@ function printRoot(
     oneIndent + fieldName + children + newLine + '}';
 }
 
-function printMutation(
-  node: RelayQuery.Mutation,
-  printerState: PrinterState
-): string {
+function printOperation(node: RelayQuery.Operation, printerState: PrinterState): string {
+  const operationKind = node.getConcreteQueryNode().kind.toLowerCase();
```

Comment:
This is reasonable and will work, but we prefer not to rely on the `kind` to correspond to a printable value (we might change it to be an integer/enum, for example). Maybe do an `instanceof` check and make the string value explicit for each subclass?